src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -60,26 +62,35 @@ public class PullRequestHandler : IHttpHandler
             context.Response.Write(HttpUtility.HtmlEncode(result));
             return;
         }
-        else
+
+        success = false;
+
+        // Check inputs to the script. PR has precedence
+        uint pr;
+        if (!string.IsNullOrEmpty(prString) && uint.TryParse(prString, out pr))
         {
-            // Check inputs to the script
-            uint pr;
-            if (!string.IsNullOrEmpty(prString) && uint.TryParse(prString, out pr))
-            {
-                success = SyncToPr(pr, result);
-            }
-            else
+            success = SyncToPr(pr, result);
+        }
+        else if (!string.IsNullOrEmpty(branchString))
+        {
+            // We only want to support limited branches as we only use this for PR purposes
+            switch (branchString)
             {
-                context.Response.Write("Invalid PR ID specified<br/>");
-                success = false;
+                case "master":
+                    success = SyncToBranch(branchString, result);
+                    break;
```

Comment:
I think that's just style preference. 

In the future this will go away once we decide what branches we want to support (probably all of them) and a better way to validate input.